recipe/meta.yaml

```diff
@@ -58,6 +58,7 @@ requirements:
     - setuptools >=31.0.1
     - toolz >=0.8.1
     - pluggy >=1.0.0
+    - importlib_resources
```

Comment:
Does this need a selector? Does it get skipped for certain Python versions that have it builtin? Or is it always needed?

Reply:
it's not needed by conda at all, this is related to tqdm being added as a dependency and https://github.com/conda-forge/tqdm-feedstock/pull/114 being open